plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties

```diff
@@ -0,0 +1,12 @@
+new.app.engine.standard.project=New App Engine Standard Project
+creating.app.engine.standard.project=Creating App Engine Standard Project
+adding.app.engine.libraries=Adding App Engine Libraries
+creating.new.app.engine.standard.project=Creating new App Engine Standard Project
+app.engine.standard.project=App Engine Standard Project
+create.app.engine.standard.project=Create a new App Engine Standard Project in the workspace.
```

Comment:
I noticed you changed the corresponding message to be Standard _Environment_ Project in #1056 for the Maven-based wizard.